src/Http/Session/Session.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Http\Session;
+
+use const PHP_SESSION_ACTIVE;
+use RuntimeException;
+use SessionHandlerInterface;
+
+final class Session
```

Comment:
We need an initializer here I think

Reply:
Yep. Usually the last item in my workflow, just in case API changes.